test.js

```diff
@@ -0,0 +1,21 @@
+const expect = require('chai').expect;
+const index = require('./index')
+
+function testNumberToPair(number, expectedMajor, expectedMinor) {
+    const pairOfColors = index.GetColorFromPairNumber(number);
+    console.log(`${number} = ${expectedMajor} ${expectedMinor}`);
+    expect(pairOfColors.major).equals(expectedMajor);
+    expect(pairOfColors.minor).equals(expectedMinor);
+}
+
+function testColorToNumber(majorColor, minorColor, expectedNumber) {
+    const pairNumber = index.GetPairNumberFromColors(majorColor, minorColor);
+    console.log(`${majorColor} ${minorColor} = ${pairNumber}`);
+    expect(pairNumber).to.equal(expectedNumber);
+}
+
+console.log("color coder");
+testNumberToPair(4, "white", "brown");
```

Comment:
The repo is in a dangerous state now... the test is happening in test.js, but the github workflow still runs index.js, which does not have the tests. If any code is changed causing failure, the CI will still pass, hiding the defect. Make sure the tests run in the CI.